### ISSUE TITLE
Make the "interval" a configurable value

### ISSUE THREAD

**Issue by slflowfoon** [valentinfrlch/ha-llmvision#146]
**Is your feature request related to a problem? Please describe.**

The core issue is that the current code uses a hardcoded, dynamic interval to determine how frequently frames are captured from a video stream. This interval is calculated based on the duration of the recording, and it's not configurable by the user.

- **Example with 10-Second Duration:** If you set the recording duration to 10 seconds, the code calculates the interval as 4 seconds. This means that it will attempt to capture a frame every 4 seconds.

- **Missed Activity:** In a fast-paced environment, a 4-second interval is too long. Crucial activity or events can happen and be missed between the captured frames.

- **Hardcoded Interval Logic:** The code uses this logic to determine the interval:
`interval = 1 if duration < 3 else 2 if duration < 10 else 4 if duration < 30 else 6 if duration < 60 else 10`

**Describe the solution you'd like**

In the services.yaml, add a section where the user can configure the interval from 0.1 to 20

**Describe alternatives you've considered**

**Additional context**

**Comment by valentinfrlch** [comment]
You are right, for fast paced environments the current logic might miss frames. 
Just for a bit of background: After frames have been captured, they are compared to each other here:
https://github.com/valentinfrlch/ha-llmvision/blob/1ab09f667d8812d3ceddfebd6842ce8cffda748b/custom_components/llmvision/media_handlers.py#L59

This filters frames below a certain movement threshold which may remove additional frames. In testing I have found that this method drastically reduces frames sent to the LLM while keeping important frames needed for context.

Still, I think your idea is good. I'll push the changes soon.
